content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -91,18 +93,69 @@ <h2 style="color: #3771e3;">Deploying your first app on Kubernetes</h2>
             <div class="col-md-8">
               <p>
                 For your first Deployment, you'll use a hello-node application packaged in a Docker container that uses NGINX to echo back all the requests. (If you didn't already try creating a hello-node application and deploying it using a container, you can do that first by following the instructions from the <a href="/docs/tutorials/hello-minikube/">Hello Minikube tutorial</a>).
-              <p>
-              
-               <p>Now that you know what Deployments are, let's go to the online tutorial and deploy our first app!</p>
+              <p>You will need to have installed kubectl as well. If you need to install it, visit <a href="/docs/tasks/tools/#kubectl">install tools</a>.</p>
+              <p>Now that you know what Deployments are, let's deploy our first app!</p>
             </div>
         </div>
         <br>
 
+        <div class="row">
+            <div class="col-md-8">
+                <h3>kubectl basics</h3>
+                <p>The common format of a kubectl command is: <code>kubectl <i>action resource</i></code></p>
+                <p>This performs the specified <em>action</em> (like create, describe or delete) on the specified <em>resource</em> (like <tt>node</tt> or <tt>deployment</tt>). You can use <code>-<span />-help</code> after the subcommand to get additional info about possible parameters (for example: <code>kubectl get nodes --help</code>).</p>
+                <p>Check that kubectl is configured to talk to your cluster, by running the <b><code>kubectl version</code></b> command.</p>
+                <p>Check that kubectl is installed and you can see both the client and the server versions.</p>
+                <p>To view the nodes in the cluster, run the <b><code>kubectl get nodes</code></b> command.</p>
+                <p>You see the available nodes. Later, Kubernetes will choose where to deploy our application based on Node available resources.</p>
+            </div>
+        </div>
         <div class="row">
             <div class="col-md-12">
-                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/" role="button">Start Interactive Tutorial <span class="btn__next">›</span></a>
+                <h3>Deploy an app</h3>
+                <p>Let’s deploy our first app on Kubernetes with the <code>kubectl create deployment</code> command. We need to provide the deployment name and app image location (include the full repository url for images hosted outside Docker hub).</p>
+                <p><b><code>kubectl create deployment kubernetes-bootcamp --image=gcr.io/google-samples/kubernetes-bootcamp:v1</code></b></p>
```

Comment:
Could these code blocks be worth formatting with a `<pre>` block, so they render similarly to the fenced code blocks that currently exist in Markdown pages?

Reply:
@sftim I've created https://github.com/kubernetes/website/issues/41625 for this discussion.